src/core/flexible_hvac.jl

```diff
@@ -50,22 +63,20 @@ be paid.
 There are two construction methods for `FlexibleHVAC`, which depend on whether or not the data was 
 loaded in from a JSON file. The issue with data from JSON is that the vector-of-vectors from the JSON 
 file must be appropriately converted to Julia Matrices. When loading in a Scenario from JSON that 
-includes a `FlexibleHVAC` model if you include the `flex_hvac_from_json` argument to the `Scenario` 
+includes a `FlexibleHVAC` model, if you include the `flex_hvac_from_json` argument to the `Scenario` 
 constructor then the conversion to Matrices will be done appropriately. For example:
 
-
-
 !!! note
     At least one of the inputs for `temperature_upper_bound_degC` or `temperature_lower_bound_degC`
     must be provided to evaluate the `FlexibleHVAC` option. For example, if only `temperature_lower_bound_degC`
     is provided then only a heating system will be evaluated. Also, the heating system will only be
     used (or purchased) if the `exogenous_inputs` lead to the temperature at the `control_node` going
     below the `temperature_lower_bound_degC`.
 
-!!! note
+!!! note # TODO: move? 
```

Comment:
@NLaws is this `ExistingChiller` note supposed to be here?

Reply:
Yes (because it is about the costs that balance the benefits of FlexibleHVAC)